components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -173,9 +173,17 @@ export class StatsService {
 
   downloadReport(format: string, filters: any[]): Observable<string> {
     const url = `${CC_API_URL}/reporting/export`;
-    const body = { type: format,
-                   filters: this.formatFilters(filters) };
+    // for export, we want to send the start_time as the beg of day of end time
+    // so we find the endtime in the filters, and then set start time to beg of that day
+    const endtime = filters.find(filter => filter['end_time']);
+    const filtersCopy = filters.map(filter => {
+      if (filter['start_time']) {
+        filter['start_time'] = moment(endtime).startOf('day');
```

Comment:
I think the below should add the 'Z' that you want, but do you want the time in UTC? 
```suggestion
       filter['start_time'] = moment(endtime).startOf('day').utc();
```

Reply:
ah, i should have edited the description - i got help from scott and the code is doing the right thing now

Reply:
the time gets set to utc during the format filters stuff

Reply:
Oh, ok. Well, I can test the code then.